docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
         GITHUB_TOKEN: ${GITHUB_TOKEN}
     container_name: myfirstcatapi-api
     ports:
-      - "${APP_PORT:-10000}:10000"
+      - "${APP_PORT:-10035}:10000"
```

Comment:
[suggestion] Instead of updating this port in here, you can also update the `.env` file (which is the recommended approach)

Reply:
done